### ISSUE TITLE
Flaky ledger api tests

### ISSUE THREAD

**Issue by oliverse-da** [digital-asset/daml#3059]
The following two tests appear to sometimes fail during multi-node (single committer/domain):

```
      // [party-id] should see exactly two transaction trees but sees 3 instead:
      "Divulgence.Divulged contracts should not be exposed by the transaction service",
      // Command interpretation error in LF-DAMLe: dependency error: couldn't find contract [contract-id]:
      "TransactionService.Expressing a non-consuming choice on a contract should not result in its archival",
      "TransactionService.Accept exercising a well-authorized multi-actor choice",
      // Alpha: either the number of archive events (25) or the number of create events (25) doesn't match the expected number of 50:
      "TransactionService.Archives should always come after creations when subscribing as more than on party",
      // Disputed: Failed to select first domain: The following parti(es) are not available on any connected domain: [party-id]:
      "TransactionService.Require only authorization of chosen branching signatory",
      "TransactionService.A transaction should be visible to a non-submitting stakeholder but its command identifier should be empty",
      "TransactionService.Not disclose create to non-chosen branching signatory",
      "TransactionService.Disclose create to the chosen branching controller",
      "TransactionService.Not disclose create to non-chosen branching controller",
      "TransactionService.Disclose create to observers",
      "TransactionService.Expose the correct stakeholders",
      "TransactionService.Reject exercising a multi-actor choice with missing authorizers",
      "TransactionService.Reject exercising a multi-actor choice with too many authorizers",
      "TransactionService.The same transaction should be served regardless of subscribing as one or multiple parties",
      "TransactionService.The same transaction trees should be served regardless of subscribing as one or multiple parties",
      "TransactionService.Expose the same events for each transaction as the output of getTransactionTrees",
      "TransactionService.Expose the same events for each transaction as the output of getTransactions",
      "TransactionService.Accept exercising a well-authorized multi-actor choice with coinciding controllers",
```

**Comment by stefanobaghino-da** [comment]
The failure message should point you at the line of code where the failure happens, if that's the case can you post it here for reference?

**Comment by stefanobaghino-da** [comment]
Looks like you are dropping all the information from the test result apart the status. [This](https://github.com/digital-asset/daml/blob/master/ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/Reporter.scala#L32-L42) is what the test tool reporter does to point at the line in the test where the issue occurred. Alternatively you can print the stacktrace of the error when it comes to you, which is more verbose but simpler. With the output you print it's impossible to tell where the error occurs.

**Comment by oliverse-da** [comment]
Fixed with 13.41.
